admin/bin/qserv-data-loader.py

```diff
@@ -52,9 +52,9 @@
 
 """
 
-#--------------------------------
+# -------------------------------
```

Comment:
I believe your editor is misinterpreting PEP-8. Do not trust it. And this is not a good reason to re-format the code because there may be other editors which say something different.

The only change to `qserv-data-loader.py` is copyright year change and few random spaces. This change should not be needed. This file should be reverted but I think you have already merged it.